lib/about_yml/schema.json

```diff
@@ -4,11 +4,11 @@
     "$schema": "http://json-schema.org/draft-04/hyper-schema",
     "type": "object",
     "properties": {
-        "name": {
+        "id": {
```

Comment:
You could also briefly support both variants in the gem, switch over the dependent projects to the new version, and then remove the old attributes in the next version of the gem.